integration/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This Dockerfile builds a base image for the certificate-transparency-go CloudBuild integration testing.
 # See https://hub.docker.com/_/golang for the set of golang base images.
-FROM golang:buster as ct_testbase
+FROM golang:buster@sha256:eb3f9ac805435c1b2c965d63ce460988e1000058e1f67881324746362baf9572 as ct_testbase
```

Comment:
Should we make this the same version as the one in `witness/Dockerfile` too, for consistency?

Reply:
Done.